### DISCUSSION TITLE
Why there is a IPsec0 interface is being created in my Server ?

### DISCUSSION THREAD

**Discussion by jayaprakashsajja** [strongswan/strongswan#638] [Q&A]
Whey there is a IPsec0 interface is being created in my Server ?
I dont know why its getting created in my strongswan which is virtual interface as soon as ipsec starts
What is the use and how to disable it if not required ?

**Comment by tobiasbrunner** [comment]
You probably have the [kernel-libipsec](https://wiki.strongswan.org/projects/strongswan/wiki/kernel-libipsec) plugin loaded. [Disable it](https://wiki.strongswan.org/projects/strongswan/wiki/PluginLoad) if you don't want that.

**Reply by jayaprakashsajja** [reply]
Thanks it worked like charm... and we see that for bursty traffic its not recommended even...
